components/raftstore/src/coprocessor/region_info_accessor.rs

```diff
@@ -766,6 +766,29 @@ impl Clone for MockRegionInfoProvider {
 }
 
 impl RegionInfoProvider for MockRegionInfoProvider {
+    fn seek_region(&self, from: &[u8], callback: SeekRegionCallback) -> Result<()> {
```

Comment:
Let's not change function ordering. This makes it hard to track code change history.